formatters/cloudevents/formatter_test.go

```diff
@@ -0,0 +1,254 @@
+package cloudevents
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/eventlogger"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFormatter_Type(t *testing.T) {
+	t.Parallel()
+	t.Run("assert-type", func(t *testing.T) {
+		assert := assert.New(t)
+		f := Formatter{}
+		assert.Equal(eventlogger.NodeTypeFormatter, f.Type())
+	})
+}
+
+func TestFormatter_Name(t *testing.T) {
+	t.Parallel()
+	t.Run("assert-name", func(t *testing.T) {
+		assert := assert.New(t)
+		f := Formatter{}
+		assert.Equal(NodeName, f.Name())
+	})
+}
+
+func TestFormatter_Reopen(t *testing.T) {
+	t.Parallel()
+	t.Run("assert-no-error", func(t *testing.T) {
+		assert := assert.New(t)
+		f := Formatter{}
+		assert.NoError(f.Reopen())
+	})
+}
+
+func TestFormatter_Process(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	testURL, err := url.Parse("https://localhost")
+	require.NoError(t, err)
+
+	now := time.Now()
+
+	tests := []struct {
```

Comment:
I've been bitten by `json.Indent` not behaving the way I'd expect, specifically, not prefixing the first line of the marshaled json with the prefix. It might be good to ensure that the text format doesn't just match the output of another marshaling command, but actually matches an expected multi-line output string with proper prefixing.

Reply:
fixed the indenting and added a test.